feat(docs): add more limitations for replicated subscriptions (#13230)
apache/pulsar: site2/docs/administration-geo.md

### Limitations

* When you enable replicated subscription, you're creating a consistent distributed snapshot to establish an association between message ids from different clusters. The snapshots are taken periodically. The default value is `1 second`. It means that a consumer failing over to a different cluster can potentially receive 1 second of duplicates. You can also configure the frequency of the snapshot in the `broker.conf` file.

> * Only markDeletePosition will be synced in replicated subscriptions while the individuallyDeletedMessages won't be synced which means the acknowledges are being done out of order cannot be synced to other clusters through replicated subscriptions.

Reviewer: ```suggestion
* Only the base line cursor position will be synced in replicated subscriptions while the individual  acknowledgments won't be synced. This that the messages acknowledged out-of-order could end up getting delivered again, in the case of a cluster failover.
```

> * Only the base line cursor position will be synced in replicated subscriptions while the individual  acknowledgments won't be synced. This that the messages acknowledged out-of-order could end up getting delivered again, in the case of a cluster failover.

Reviewer 2: ```suggestion
* Only the base line cursor position is synced in replicated subscriptions while the individual acknowledgments are not synced. This means the messages acknowledged out-of-order could end up getting delivered again, in the case of a cluster failover.
```

- Always use the present tense in technical writing by default. 
- Use present tense if you are covering facts that were, are, and forever shall be true.
- Only use past or future tenses when you’re actually writing about something that happened in the past or will happen in the future.
